### ISSUE TITLE
Arguments for `backoff-ms` in `with-retry` throw spec error

### ISSUE THREAD

**Issue by borkdude** [sunng87/diehard#14]
In the docs for `with-retry`:

```
:backoff-ms specify a vector [initial-delay-ms max-delay-ms multiplier]
```

When supplying e.g. `[2000 60000 2]` I get a spec error:
```
Invalid input
   {:clojure.spec.alpha/problems
    ({:path [:backoff-ms :single],
      :pred clojure.core/int?,
      :val [2000 60000 2],
      :via [:retry/retry-block :retry/backoff-ms],
      :in [:backoff-ms]}
     {:path [:backoff-ms :tuple],
      :pred (clojure.core/= (clojure.core/count %) 2),
      :val [2000 60000 2],
      :via [:retry/retry-block :retry/backoff-ms],
      :in [:backoff-ms]}),
    :clojure.spec.alpha/spec :retry/retry-block,
    :clojure.spec.alpha/value
    {:on-retry
     #function[.../fn--79386],
     :retry-on java.lang.Exception,
     :max-retries 10,
     :backoff-ms [2000 60000 2]}}
```

Apparently the spec contradicts the documentation:

https://github.com/sunng87/diehard/blob/3a97826dc1f6cbab2e1f3910acb43c824cba86b9/src/diehard/spec.clj#L38

**Comment by sunng87** [comment]
Thank you for reporting. Yes the issue is related to this spec, which didn't count multiplier. I will fix and release a new version later today.

**Comment by sunng87** [comment]
Fixed in 0.7.1

**Comment by borkdude** [comment]
Thanks!
